crawl-ref/source/spl-util.cc

```diff
@@ -1625,7 +1625,6 @@ bool spell_no_hostile_in_range(spell_type spell)
         return true;
 
     const int range = calc_spell_range(spell, 0);
-    const int minRange = get_dist_to_nearest_monster();
```

Comment:
Why are you moving this, then defining it twice (as const)? I'd suggest leaving it where it is, and not repeating ourselves.

Reply:
`get_dist_to_nearest_monster` is a fairly expensive function to call and most spells don't use its result so I moved it to the two places that use it. I've undone this change as it had nothing to do with OTR's inconsistency.